### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by ChriMarMe**
Improved some error msgs.
Removed some magic numbers

### CODE REVIEW

File: vendor/github.com/fearful-symmetry/gomsr/go.mod [hunk 1]
```diff
@@ -1 +1,3 @@
 module github.com/fearful-symmetry/gomsr
+
+go 1.13
```
Comment (PatrickRudolph) [inline vendor/github.com/fearful-symmetry/gomsr/go.mod]: seems unrelated

Comment (ChriMarMe) [reply vendor/github.com/fearful-symmetry/gomsr/go.mod]: deleted. maybe a relic of other branch, i cant tell.

File: vendor/github.com/fearful-symmetry/gomsr/go.mod [hunk 2]
```diff
@@ -1 +0,0 @@
-module github.com/fearful-symmetry/gomsr
```
Comment (PatrickRudolph) [inline vendor/github.com/fearful-symmetry/gomsr/go.mod]: unrelated change

Comment (ChriMarMe) [reply vendor/github.com/fearful-symmetry/gomsr/go.mod]: deleted
